README.md

```diff
@@ -39,3 +39,11 @@ For help setting up and using the Qiskit Code Assistant read [GETTING_STARTED](G
 ## Contributing
 
 To learn more about contributing or installing locally read [CONTRIBUTING](CONTRIBUTING.md).
+
+## Qiskit Code Assistant Terms of use
+
+- Terms of use: [https://quantum.ibm.com/terms](https://quantum.ibm.com/terms)
+- Privacy policy: [https://quantum.ibm.com/terms/privacy](https://quantum.ibm.com/terms/privacy)
+- Cloud Services Agreement [https://www.ibm.com/support/customer/csol/terms/?id=Z126-6304&cc=us&lc=en](https://www.ibm.com/support/customer/csol/terms/?id=Z126-6304&cc=us&lc=en)
+- IBM Cloud Service Description [https://www.ibm.com/support/customer/csol/terms/?id=i126-6605&lc=en](https://www.ibm.com/support/customer/csol/terms/?id=i126-6605&lc=en)
+- [End User License Agreement (EULA)](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab/blob/main/docs/EULA.md) acceptance required before starting to use the model acceptance required before starting to use the model
```

Comment:
since this is a GitHub file, unlike `README-PyPi.md`, we can use relative links here

Reply:
Done in https://github.com/Qiskit/qiskit-code-assistant-jupyterlab/pull/4/commits/d5537c832214230652fa5d9d36b7c4c8278ddecb